### PR TITLE
#165020160 setup SASS processing on the client

### PR DESCRIPTION
**Pull request by fridolinho**
### What does this PR do?
Setup sass on the client
### Description of Task to be completed?
    Install package dependencies

- sass-loader

- node-sass

- webpack

  
### How should this be manually tested?
- Create a new sass file in the components folder and add styles
- Import it in the `app.js` like this `import  './nameoffile.scss'`

### What are the relevant pivotal tracker stories?
[#165020160 setup SASS processing on the client](https://www.pivotaltracker.com/story/show/165020160)
### Screenshots (if appropriate)
N/A
### Questions:
N/A

### CODE REVIEW

**Review by avpaul — APPROVED** [review]
Works as expected!

**Comment by ezkemboi** [comment]
@fridolinho  please rebase the latest changes and fix conflicts.

File: webpack.config.babel.js [hunk 1]
```diff
@@ -16,6 +16,14 @@ const config = {
   stats: 'errors-only',
   module: {
     rules: [
+      {
```
Comment (ezkemboi) [inline webpack.config.babel.js:19]: Good work @fridolinho.

**Review by ezkemboi — APPROVED** [review]
Works as expected but there is need to fix conflicts and rebase the latest changes in the develop branch.

**Pull request merged**